### ISSUE TITLE
Potential of frying my m1 mac while following the guide?

### ISSUE THREAD

**Issue by haznai** [asahi-guix/channel#2]
As i'm extremely new to asahi linux, guix and installing non-macOs systems on my mac, i'd love to hear your thoughts on the potential of fryin' my mac while trying to install asahi guix.

this question comes from a place of genuine interest in the project ☺️

**Comment by haznai** [comment]
also, extra question: what is the current support of the latest asahi linux, and what are your guesses on how sustainable an asahi guix installation _can_ be on a m1 mac

**Comment by r0man** [comment]
Hi @haznai, 

sorry I just saw this now. Thanks for the interest. Regarding your questions:

- I would not recommend using this project if you are new to all of this, unless you want to get your hands dirty
- That said, I'm using it since quite some time and it works "good enough" for me
- Sound is working since yesterday
- There are occasional "breakages" with Mesa, but this is probably me not knowing what I do
- You will fry your Mac for sure, not in the sense of breaking it, but you have to build A LOT of packages. This is hopefully solved once the core-updates branch of Guix is merged into master.
- This repo tries to be "up to date" with the "latest" Asahi Linux

I hope that helps.
